cmd/prettylog/prettylog.go

```diff
@@ -19,7 +19,18 @@ func main() {
 	writer := zerolog.NewConsoleWriter()
 
 	if isInputFromPipe() {
-		_, _ = io.Copy(writer, os.Stdin)
+		scanner := bufio.NewScanner(os.Stdin)
+		for scanner.Scan() {
+			bytesToWrite := scanner.Bytes()
+			_, err := writer.Write(bytesToWrite)
+			if err != nil {
+				if errors.Is(err, io.EOF) {
+					break
+				}
+
+				fmt.Printf("%s\n", bytesToWrite)
+			}
+		}
```

Comment:
Could you please make this a function and reuse it for the code below (for file arg)?

Reply:
Yeah, I thought about this, but it's slightly different in terms of how the errors are treated. What do you think would be the best way to do in both cases if error occurs, crash the app and display the error message, or display the output as is?

Reply:
Showing the line as-is is probably sensible for such a tool.